src/Containers/ContributionForm/ContributionForm.js

```diff
@@ -121,127 +122,165 @@ function ContributionForm() {
-        />
-        {profession === 'mhp' &&
-          <input
-          type='text'
-          placeholder='Cost'
-          name='cost'
-          value={cost}
-          onChange={event => handleChange(setCost, event)}
-          />
-        }
-        <input
-          type='text'
-          placeholder='Street'
-          name='street'
-          value={street}
-          onChange={event => handleChange(setStreet, event)}
-        />
-        <input
-          type='text'
-          placeholder='Unit'
-          name='unit'
-          value={unit}
-          onChange={event => handleChange(setUnit, event)}
-        />
-        <input
-          type='text'
-          placeholder='City'
-          name='city'
-          value={city}
-          onChange={event => handleChange(setCity, event)}
-        />
-        <DropDown
-          options={[
-            'Alabama', 'Alaska', 'Arizona', 'Arkansas', 'California', 'Colorado', 'Connecticut', 'Delaware', 'Florida', 'Georgia',
-            'Hawaii', 'Idaho', 'Illinois', 'Indiana', 'Iowa', 'Kansas', 'Kentucky', 'Louisiana', 'Maine', 'Maryland',
-            'Massachusetts', 'Michigan', 'Minnesota', 'Mississippi', 'Missouri', 'Montana', 'Nebraska', 'Nevada', 'New Hampshire', 'New Jersey',
-            'New Mexico', 'New York', 'North Carolina', 'North Dakota', 'Ohio', 'Oklahoma', 'Oregon', 'Pennsylvania', 'Rhode Island', 'South Carolina',
-            'South Dakota', 'Tennessee', 'Texas', 'Utah', 'Vermont', 'Virginia', 'Washington', 'West Virginia', 'Wisconsin', 'Wyoming'
-          ]}
-          label='state'
-          value={state}
-          handleChange={event => handleChange(setState, event)}
-        />
-        <input
-          type='text'
-          placeholder='Zip Code'
-          name='zip'
-          value={zip}
-          onChange={event => handleChange(setZip, event)}
-        />
-        <input
-          type='text'
-          placeholder='Phone Number'
-          name='phone'
-          value={phone}
-          onChange={event => handleChange(setPhone, event)}
-        />
-        <button className='contribution-button' onClick={(event) => handleClick(event)}>Submit</button>
-        {error &&
-          <Error error={error} />
-        }
-      </form>
+  const { darkMode, light, dark } = useContext(ThemeContext);
+  const theme = darkMode ? dark : light;
+
+    return (
+      <>
+        <section className='theme-wrapper' style={{ color: theme.color, background: theme.background }}>
+
+          <form>
+            <section className='contribution-title'>
+
+              <h2>Contribution Form</h2>
+
+              {submissionMessage &&
+              <h2>{submissionMessage}</h2>}
+            </section>
+
+              <section className='contribution-form'>
+                <article className='form-name-wrapper'>
+                  <h3 className='form-directions'>Please Fill In Your Provider's First and Last Name</h3>
```

Comment:
The instructions through this form are good! I'm sure this will help cut down on user confusion